k5/js/main.js

```diff
@@ -66,15 +66,15 @@ function performAfterSort(sorter) {
   reloadDisplay();
 }
 
-// TSVをデータに変換
+// TSVをデータに変換しチェックボックス配置のための空のカラムの追加
 function dataStore(data) {
   cachedData = data;
 
   let hospitals = [];
 
   data.forEach(function (row) {
     if (dataHeaders.length == 0) {
-      dataHeaders = row;
+      dataHeaders = row.unshift("viz");
```

Comment:
テーブルのヘッダにある名称が出なくなってました

```suggestion
      row.unshift("viz");
      dataHeaders = row;
```